docs/01_introduction.md

```diff
@@ -1,48 +1,46 @@
 ## What is EasyBuild?
 
 [EasyBuild](http://easybuilders.github.io/easybuild/) is a **software build and installation framework**
-that allows you to manage (scientific) software on [**High Performance Computing (HPC)**](https://en.wikipedia.org/wiki/Supercomputer)
-systems in an efficient way, without compromising on performance.
+that manages (scientific) software in an efficient way, without compromises on performance. Easybuild is especially tailored towards [**High Performance Computing (HPC)**](https://en.wikipedia.org/wiki/Supercomputer)
+ environments.
 
 
 ### *Elevator pitch*
 
 <a href="http://geek-and-poke.com/geekandpoke/2010/5/14/how-to-become-invaluable.html"><img src="https://boegel.github.io/easybuild-tutorial/img/geek-and-poke-invaluable.png" style="float:right" width="350px"/></a>
 
-EasyBuild intends to relieve HPC user support teams of the ubiquitous burden of manually executing
-software installation procedures, and **facilitate providing a consistent and well performing
+EasyBuild intends to relieve HPC support teams from manually managing
+software installations while at the same time **providing a consistent and well performing
 scientific software stack** to end users of HPC systems.
 
-It serves as a **uniform interface for installing scientific software**,
-and saves valuable time (and frustration) by **automating** tedious, boring and repititive tasks.
-As such, it has grown out to be an **expert system** for software installation on HPC systems.
```

Comment:
Did you mean to kick out the `expert system` line?

Reply:
From our conversation on Slack:

@terjekv:
It’s a matter of audience. I think it should be kept if the aim is to sell it to high end HPC sysadmins, but I fear it can scare off the lower, smaller, rung of admins.

@boegel  
hmm, ok, makes sense
I guess it depends on how you interpret "expert system". To me, that's something built by experts, not for experts, but I see your point.

@terjekv:
My thought was those who know won’t really care and those who don’t might be intimidated.